src/ray/common/cgroup/cgroup_setup.cc

```diff
@@ -176,6 +227,38 @@ bool SetupCgroupsPreparation(const std::string &node_id) {
   return true;
 }
 
+void CleanupCgroupForNode(const std::string &node_id) {
```

Comment:
in line with the suggestions above, each raylet manages only a single node so there's really no reason we should need to pass the `node_id` in beyond initializing the cgroup manager obj

Reply:
I would actually consider it an abstraction leak that the cgroup manager knows anything about a "node" at all

Reply:
I wrap all related functions into a class, so no need to pass arguments any more.